ms_active_directory/core/ad_session.py

```diff
@@ -3899,12 +3899,13 @@ def _move_or_rename_object_by_dn(self, ad_object: Union[str, ADObject], new_pare
         object_dn = ad_object.distinguished_name
         object_name = ad_object.get(ldap_constants.AD_ATTRIBUTE_COMMON_NAME, unpack_one_item_lists=True)
         new_name = new_name if new_name else object_name
+        new_rdn = 'cn=' + new_name
```

Comment:
could you make this only add the `cn=` prefix if it's not already there? that way we maintain backwards compatibility for people already working around this?

Reply:
I've pushed a change. Not sure if you'd prefer an expanded `if` statement for it as its possibly a lot for one line.